src/Controls/src/Core/Border.cs

```diff
@@ -145,6 +145,18 @@ public float[] StrokeDashPattern
 
 		IView IContentView.PresentedContent => Content;
 
+		protected override void OnBindingContextChanged()
+		{
+			base.OnBindingContextChanged();
+
+			BindableObject content = Content as BindableObject;
+
+			if (content != null)
```

Comment:
Is this accounted for somewhere? maybe the base `BindableObject`.
```csharp
var border = Border();

var bindingContext = new object();
border.BindingContext = bindingContext;

var content = new ContentView();
border.Content = content;

Assert.True(content.BindingContext == bindingContext);
Assert.True(content.BindingContext == border.BindingContext);
```

Reply:
I believe it is this test: https://github.com/dotnet/maui/pull/3315/files#diff-a186d7040a43a5d73d4827e554cf6c9ce961506def6072d802032628d4d14d5fR19